org.gabmus.razercommander.json

```diff
@@ -0,0 +1,104 @@
+{
+    "app-id": "org.gabmus.razercommander",
+    "command": "razercommander",
+    "branch": "master",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "razercommander",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=system-bus",
+        "--share=network",
+
+        "--talk-name=org.gnome.SettingsDaemon",
+        "--talk-name=org.razer",
+
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--filesystem=~/.config/razercommander:rw",
+        "--filesystem=~/.config:rw",
+        "--talk-name=ca.desrt.dconf",
+
+        "--env=SSL_CERT_FILE=/etc/ssl/certs/ca-certificates.crt"
+    ],
+    "modules": [
+        {
+            "name": "razercommander",
+            "buildsystem": "meson",
+            "builddir": true,
+            "sources": [{
+                "type": "archive",
+                "url": "https://github.com/GabMus/razerCommander/archive/1.2.1.2.tar.gz",
+                "sha256": "1fe593ec5b656a4cc58be983470dc6fb9332a15eaec42e21e66bb6c264e3818a"
+            }]
+        },
+        {
+            "name": "python-dbus",
+            "make-install-args": ["prefix=/app"],
+            "config-opts": ["prefix=/app"],
+            "make-args": ["prefix=/app"],
+            "build-options": {
+                "env": {
+                    "PYTHON": "python3"
+                }
+            },
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://dbus.freedesktop.org/releases/dbus-python/dbus-python-1.2.4.tar.gz",
+                    "sha256": "e2f1d6871f74fba23652e51d10873e54f71adab0525833c19bad9e99b1b2f9cc"
+                }
+            ]
+        },
+        {
+            "name": "python-numpy",
+            "buildsystem": "simple",
+            "ensure-writable": [
+                "/lib/python3.6/site-packages/easy-install.pth",
+                "/lib/python3.6/site-packages/setuptools.pth",
+                "/app/lib/python3.6/site-packages/easy-install.pth",
+                "/app/lib/python3.6/site-packages/setuptools.pth"
+            ],
+            "build-commands": [
+                "python3 setup.py build",
+                "python3 setup.py install --prefix=/app"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/numpy/numpy/releases/download/v1.13.3/numpy-1.13.3.tar.gz",
+                    "sha256": "4c6b4eef790528bebb7ec9590d74cc193868940fe68e4109a91c196df72d8094"
+                }
+            ]
+        },
+        {
+            "name": "python-openrazer",
+            "buildsystem": "simple",
+            "ensure-writable": [
+                "/lib/python3.6/site-packages/easy-install.pth",
```

Comment:
Use a * in the path to be a little more robust here. Example: https://github.com/flathub/work.openpaper.Paperwork/blob/742800038615e1b3552f828b327a9275b8db6162/work.openpaper.Paperwork.json#L42

Reply:
Done! What about those failing checks?